### DISCUSSION TITLE
Import Modpacks in The terminal Version for Mac

### DISCUSSION THREAD

**Discussion by VaproI** [KSP-CKAN/CKAN#4180] [Q&A] [answered]
Im trying to import a modpack for  KSP, but when I try it does not recognize any files unless they are .zip
![Screenshot 2024-09-09 at 17 13 05](https://github.com/user-attachments/assets/86b4f846-b1b0-4497-9eb9-3b6517c5048e)
![Screenshot 2024-09-09 at 17 13 21](https://github.com/user-attachments/assets/a69a2cc0-2db3-450d-8b39-161881e074cd)

**Comment by HebaruSan** [comment, marked as answer]
Hi @VaproI, that feature isn't for installing modpacks. It's for importing manually-downloaded ZIP files.

Modpack installation was added to the ConsoleUI in KSP-CKAN/CKAN#4103. It'll be included in the next release of CKAN.

**Reply by VaproI** [reply]
Oh so it is not possible yet?

**Reply by HebaruSan** [reply]
It's not possible in ConsoleUI in the current release.
If you know how to use the CKAN command line, you can do it like this:

```
mono ckan install -c AOMODS1.ckan
```

Actually I think I might have seen people using `CKAN` instead of `mono ckan` on Mac to take advantage of the startup script that we wrote:

```
CKAN install -c AOMODS1.ckan
```

**Reply by VaproI** [reply]
no sadly I don't really know how to operate the terminal but thank you anyways.
